docs/BulkExport.md

```diff
@@ -0,0 +1,19 @@
+# Bulk Export
+
+This feature allows data from the FHIR server to be exported. More detail can be found in the [spec](https://github.com/HL7/bulk-data/blob/master/spec/export/index.md).
+
+The feature is currently under preview and is turned off by default. To enable the feature, update the `FhirServer:Operations:Export:Enabled` setting to be true.
+
+## Specifying destination
+
+One customization that we have done in addition to the spec is requiring the destination information to be supplied. The destination type can be specified by using `_destinationType` query parameter and any destination specific settings can be specified as BASE64 encoded string by using `_destinationConnectionSettings` query parameter.
```

Comment:
Note this is a breaking change from the specification if it adds a new, required parameter. We should call it out as such (and ideally document that we plan to support a default destination to maintain compatibility).

Reply:
Sure I can make it more clear.